freak/post_processing_freak.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
```

Comment:
I think freak is written in python2. Both tools should be written to have the same requirements where possible. Also, this shebang should be removed for portability and usability in galaxy instances.

Reply:
Hmm, would you like the code to be python2 compatible? I haven't been checking for backwards compatibility.

Reply:
Yes, python2 compatible please. The freak tool and the postprocessing will rely on the same conda env, same python version, same package dependencies, etc.